collectors/wallet_collector.go

```diff
@@ -198,17 +198,21 @@ func (u *WalletCollector) Collect(ch chan<- prometheus.Metric) {
 		path := account.GetDerivationPath()
 
 		// internal key count.
-		ch <- prometheus.MustNewConstMetric(
-			u.keyCountDesc, prometheus.CounterValue,
-			float64(account.InternalKeyCount),
-			name, addrType, path, "internal",
-		)
+		if count := account.InternalKeyCount; count > 0 {
```

Comment:
Why not just `if account.InternalKeyCount {`?

Reply:
that would be a non-boolean condition in if statement

Reply:
Yes, you're right. Sorry, I meant `if account.InternalKeyCount > 0 {`

Reply:
because I use `count` as an alias, which is shorter to type, and if for w/e reason the variable name changes (unlikely though), I don't have to change it in 2+ places. But I can change it you want.

Reply:
Golang code should have the focus on readability, not less typing work. This on first glance almost looks like a `for` loop. And it's AFAIK not very common practice to use aliasing within an `if` clause.
Also, the diff gets slightly more complex to review (since more lines contain non-whitespace changes).

I'm aware we're splitting hairs here. This is mostly meant as a tip on how to make reviewer's lives easier.

Reply:
fair enough, thanks for explaining. Will update